src/System/Console/Hawk/Context/Base.hs

```diff
@@ -58,8 +58,8 @@ newContext paths = do
     userPrelude <- readUserPrelude originalFile
     lift $ createDirectoryIfMissing True cacheDir
     compileUserPrelude originalFile canonicalFile userPrelude
-    
-    return $ Context
+
+    return Context
```

Comment:
I am again surprised that no parentheses are needed with record syntax! Good to know!